extension/addreactionparticipants.js

```diff
@@ -0,0 +1,60 @@
+const addReactionParticipants = currentUser => {
+	$('.comment-reactions').each((index, reactionsContainer) => {
```

Comment:
What do you think about having `addReactionParticipants` accept an optional comment container/id, or something along those lines? Right now, it looks like, when a user reacts (line 13), we're going to iterate over and re-render every reaction bar for each comment on the page. Knowing which comment is the target would eliminate that.